src/devtools/client/debugger/src/components/WelcomeBox.tsx

```diff
@@ -2,69 +2,225 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at <http://mozilla.org/MPL/2.0/>. */
 
-import React from "react";
```

Comment:
The changes to this component will be reverted before the PR is marked _ready_ for review. I'm just using it as a testing ground.